app/datatables/repository_datatable.rb

```diff
@@ -230,12 +232,12 @@ def filter_records(repo_rows)
 
   # Override default sort method if needed
   def sort_records(records)
-    if params[:order].present? && params[:order].length == 1
-      if sort_column(params[:order].values[0]) == ASSIGNED_SORT_COL
+    if order_params.present?
```

Comment:
as above it always evaluate true so why we need this? Before that was also a :banana: before params[:order].length == 1 
hmm...

Reply:
Ok, removed that check